content/en/blog/2024/cve-2024-36129/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Security vulnerability in confighttp (CVE-2024-36129)
+linkTitle: Security vulnerability in confighttp
+date: 2024-06-05
+author:
+  '[Juraci Paixão Kröhling](https://github.com/jpkrohling) (OpenTelemetry,
+  Grafana Labs), [Pablo Baeyens](https://github.com/mx-psi) (OpenTelemetry,
+  Datadog)'
+cSpell:ignore: confighttp Baeyens OSTIF zstd configgrpc Miroslav Stampar
+---
+
+On our path toward graduation, the OpenTelemetry project is currently undergoing
+a security audit sponsored by the [CNCF](https://www.cncf.io/), facilitated by
+[OSTIF](https://ostif.org/), and performed by
+[7ASecurity](https://7asecurity.com/). During this process, we have received a
+few ideas about things that we could do better, like using specific compiler
+flags when preparing our [OpenTelemetry Collector
+binaries](https://github.com/open-telemetry/opentelemetry-collector-releases/releases).
+On 31 May 2024, we received [a more serious
+report](https://github.com/open-telemetry/opentelemetry-collector/security/advisories/GHSA-c74f-6mfw-mm4v):
+a malicious user could cause a denial of service (DoS) when using a specially
+crafted HTTP or gRPC request. The advisory was assigned the following CVE
+identifier: [CVE-2024-36129](https://nvd.nist.gov/vuln/detail/CVE-2024-36129).
```

Comment:
This one is not available yet :/ We can remove now and add later to work around CI:

```suggestion
identifier: CVE-2024-36129.
```